doc/developer-guide/plugins/example-plugins/index.en.rst

```diff
@@ -48,9 +48,9 @@ understand the following topics:
 -  Working with HTTP header functions
 
 The two sample plugins discussed in this chapter are ``denylist_1.cc``
-and ``basic_auth.cc``. To build and install the example plugins use ::
-
-   ./configure --enable-example-plugins
+and ``basic_auth.cc``. To make this two plugins available, you must enable experimental plugins when
+building |TS| by passing the ``-DBUILD_EXPERIMENTAL_PLUGINS=ON`` to the ``cmake`` command
```

Comment:
This looks an "example" plugin. It should be `-DENABLE_EXAMPLE=ON`?

Reply:
you are totally right! :cry:  Thanks!!